### PR TITLE
logger implemented in request api

### PR DESCRIPTION
**Pull request by JuanCarlosMartinez4**

### CODE REVIEW

**Review by Amet015 — APPROVED** [review]

File: core/api/request_api/RequestApi.py [hunk 1]
```diff
@@ -1,6 +1,9 @@
 import json
 import re
+from http.client import HTTPConnection
+
 import requests
+import logging
 
 from core.utils.EnvironmentConfiguration import EnvironmentConfiguration
```
Comment (Enrique-C) [inline core/api/request_api/RequestApi.py]: Review the imports

File: core/api/request_api/RequestApi.py [hunk 2]
```diff
@@ -41,6 +44,15 @@ def do_request(self, http_type, input_endpoint, data_table, id_dictionary):
         url = self.config.get_config_file()['base_uri'] + self.replace_variables(input_endpoint, id_dictionary)
         HEADERS = {'content-type': 'application/json'}
 
+        HTTPConnection.debuglevel = 1
+        logger = logging.getLogger()
+        logger.setLevel(logging.DEBUG)
+        fh = logging.FileHandler("logs/myapp.log")
```
Comment (Enrique-C) [inline core/api/request_api/RequestApi.py]: Would you use environment var

File: logs/logFile.log [hunk 3]
```diff
@@ -0,0 +1,24 @@
+2020-02-04 12:50:23,345 - 1240 - urllib3.connectionpool - DEBUG - Starting new HTTPS connection (1): api.trello.com:443
```
Comment (Enrique-C) [inline logs/logFile.log]: Review this file please

File: config.json [hunk 4]
```diff
@@ -3,5 +3,6 @@
   "folder_json_path": "trello/jsonfiles/",
   "key": "",
   "token": "",
-  "email" : ""
+  "email" : "",
```
Comment (Enrique-C) [inline config.json]: review the indentation please

**Review by Enrique-C — APPROVED** [review]

**Pull request merged**